package-lock.json

```diff
@@ -1,8 +1,4175 @@
 {
   "name": "@cloudant/couchbackup",
   "version": "2.9.8-SNAPSHOT",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
We've had issues in the past with the peer dependencies working correctly in Node 14/NPM 6 with the v2 lockfiles from newer versions. I think you'll need to regenerate this with the older version.

Reply:
thanks for spotting, done in 66a8842